zappa/handler.py

```diff
@@ -129,14 +129,14 @@ def handler(self, event, context):
                 response = Response.from_app(app, environ)
 
                 # This is the object we're going to return.
-                zappa_returndict = dict()
+                # Pack the WSGI response into our special dictionary.
+                zappa_returndict = {header_name: header_value
```

Comment:
As long as we are making tiny refactor PRs, might as well just do `dict(response.headers)`